### ISSUE TITLE
disable b-emlog.txt creation; command-line option to enable

### ISSUE THREAD

**Issue by EndeavourAccuracy** [stardot/b-em#55]
I suggest disabling the creation of b-emlog.txt, and then add a command-line option to enable its creation for those who really want it.

**Comment by SteveFosdick** [comment]
While I get round to this you can disable the log file from the config file.  Edit b-em.cfg and if there isn't a section [logging] add one and then add to_file= to it, like this:

```
[logging]
to_file=
```

The value you would be setting to empty string is not the filename but the set of message levels sent to the file - if that set is empty then it doesn't open the file.  You can also control which messages result in dialogue boxes and which are sent to stderr with similar config keys of to_stderr and to_msgbox and levels of FATAL,ERROR,WARNING,INFO,DEBUG

The default is equivalent to:
```
[logging]
to_file=FATAL,ERROR,WARNING,INFO,DEBUG
to_stderr=FATAL,ERROR,WARNING
to_msgbox=FATAL,ERROR
```

**Comment by EndeavourAccuracy** [comment]
That would be a good solution. I'm not sure what you mean by "While I get round tot his". You mean it's on the TODO list? It's doesn't seem to work yet, but I did just now find code that seems to detect these settings, in log_open() of logging.c.

**Comment by SteveFosdick** [comment]
Try
```
[logging]
to_file=none
```
it seems the empty string is being interpreted by the Allegro config routine as undefined and therefore it is using the default.  The string could be anything but "none" is pretty plain in what it means.  What us needed is that none of the know log levels appear in that value.

If you're happy with this as the solution then I need to do nothing further, assuming what I have suggested works for you this time.  I did just test it and it works for me.
